File: python-training/age.js

//come back in x year(s)
var current_year = new Date();
var current_year = current_year.getFullYear();

var username = prompt("Enter name: ");
var year_of_birth = prompt("Enter year of birth: ");
var age = current_year - year_of_birth;

if (age >= 18){
    window.alert("You are eligible, please apply.");
}else {
        var ok_year = 18 - age;
    window.alert("You are not eligible, come back in " + ok_year + " year(s)!");
}

/// Come back in year(x)
var current_year = new Date();
var current_year = current_year.getFullYear();

var username = prompt("Enter name: ");
var year_of_birth = prompt("Enter year of birth: ");
var age = current_year - year_of_birth;

if (age >= 18){
    window.alert("You are eligible, please apply.");
}else {
		var ok_year = 18 - age;
		var ok_year = current_year + ok_year;
    window.alert("You are not eligible, come back in " + ok_year +"!");
}

File: python-training/code-drafts/age.py
import datetime  # import datetime

# get current datetime from datetime library
t = datetime.datetime.now()
current_year = t.year  # get only the year

username = input('Enter Name: ')
year_of_birth = input("Enter Year of Birth: ")
year_of_birth = int(year_of_birth)
age = current_year - year_of_birth
if age > 18:
    flash_message = "{} you are {} and eligible, kindly apply."
    print(flash_message.format(username,age))
else:
    ok_year = 18 - age
    ok_year = current_year + ok_year
    flash_message = "You are {} and not eligible, for the job now, apply in {}."
    print(flash_message.format(age, ok_year))